### ISSUE TITLE
Integrated Authentication Sites not in Firefox 52.0.2 Tools.

### ISSUE THREAD

**Issue by plexos** [tillig/FirefoxNtlmAuth#26]
This was not in Tools because I had "Never Remember History" set. It appeared after setting Preferences: Privacy to "Remember History".

**Comment by tillig** [comment]
I'm curious if this has something to do with the fact I'm using [what they now call] the ["legacy add-on SDK"](https://developer.mozilla.org/en-US/Add-ons/SDK). I gather I'm going to have to migrate to the new ["WebExtensions" mechanism](https://developer.mozilla.org/en-US/Add-ons/WebExtensions). Which is sort of frustrating because this is the second time in the last couple of years that they've totally deprecated their plugin SDK in favor of something new.

**Comment by tillig** [comment]
It appears that I may be stuck here and the whole plugin will end up being deprecated/unsupported by the end of 2017. Per the [migration document on updating to WebExtensions settings](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/Comparison_with_the_Add-on_SDK#Settings):

> Note that WebExtensions does not give you access to the browser's own preferences (that is, the preferences exposed in the SDK by `preferences/service`).

The whole point of my extension is to modify certain browser preferences in that location.

[I found a StackOverflow question on this exact topic](http://stackoverflow.com/questions/37241450/how-to-change-firefox-preferences-aboutconfig-using-the-new-webextensions-api) and there's a Firefox [FAQ on this](https://wiki.mozilla.org/WebExtensions/FAQ#Will_I_have_access_to_about:config_or_the_preferences.3F).

Basically... sounds like I'm hosed. I'll keep watching it, but there's a pretty low likelihood that I can do anything about it.

In the meantime, I'll update the wiki docs to tell folks that "Never Remember History" won't work with this extension.
